### DISCUSSION TITLE
Let me nest custom sources

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#11389] [Ideas]
I'd like to be able to create custom sources under existing sources - this helps keep the element index sidebar from getting too cluttered. For example:

```
Blog Posts (section)
-> Posts without images (custom source)
Events (section)
-> 2022 Events (custom source)
-> 2021 Events (custom source)
-> 2020 Events (custom source)
```

**Comment by kylecotter** [comment]
We've resorted to some unicode arrow characters to fake this, but would love to see this built in too!

**Reply by Mosnar** [reply]
I ended up using these characters for now: `├` `└ `

This is the result:
![image](https://user-images.githubusercontent.com/3810939/175941379-65a828f5-12ca-4bf7-ad59-868dbbf83dd6.png)
